cluster-autoscaler/cloudprovider/openstack/examples/os-autoscaler-deployment-master.yaml

```diff
@@ -0,0 +1,51 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: os-autoscaler
+  namespace: kube-system
+  labels:
+    app: autoscaler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: autoscaler
+  template:
+    metadata:
+      namespace: kube-system
+      labels:
+        app: autoscaler
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      securityContext:
+        runAsUser: 1001
+      tolerations:
+        - key: node.cloudprovider.kubernetes.io/uninitialized
+          value: "true"
+          effect: NoSchedule
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+      serviceAccountName: os-autoscaler-account
+      containers:
+        - name: os-autoscaler
+          image: thartland/os-autoscaler:v1.0
+          imagePullPolicy: Always
+          command:
+            - ./cluster-autoscaler
+            - --alsologtostderr
+            - --cluster-name=scaler-01
+            - --cloud-config=/config/cloud-config
+            - --cloud-provider=openstack
+            - --nodes=1:10:DefaultNodeGroup
+            - --scale-down-unneeded-time=30s
```

Comment:
Consider dropping these three --scale params from the merge request before merging (i guess they are used for testing).

Reply:
Will do.

Reply:
Done in https://github.com/kubernetes/autoscaler/pull/1690/commits/1e3f725274f112baed9c1608fca60fa9cd7f0daf.